assets/js/modules/analytics/components/setup/SetupFormUA.stories.js

```diff
@@ -0,0 +1,91 @@
+/**
+ * SetupFormUA component stories.
+ *
+ * Site Kit by Google, Copyright 2021 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Internal dependencies
+ */
+import { STORE_NAME } from '../../datastore/constants';
+import { MODULES_ANALYTICS_4 } from '../../../analytics-4/datastore/constants';
+import { provideModules, provideModuleRegistrations } from '../../../../../../tests/js/utils';
+import { enabledFeatures } from '../../../../features';
+import ModuleSetup from '../../../../components/setup/ModuleSetup';
+import WithRegistrySetup from '../../../../../../tests/js/WithRegistrySetup';
+import * as fixtures from '../../datastore/__fixtures__';
+
+export const Ready = () => <ModuleSetup moduleSlug="analytics" />;
+Ready.storyName = 'SetupFormUA';
+Ready.decorators = [
+	( Story ) => {
+		const setupRegistry = ( registry ) => {
+			const { accounts, properties, profiles } = fixtures.accountsPropertiesProfiles;
+			/* eslint-disable sitekit/acronym-case */
+			const accountID = properties[ 0 ].accountId;
+			const propertyID = profiles[ 0 ].webPropertyId;
+
+			registry.dispatch( STORE_NAME ).receiveGetSettings( {} );
+			registry.dispatch( STORE_NAME ).receiveGetAccounts( accounts );
+			registry.dispatch( STORE_NAME ).finishResolution( 'getAccounts', [] );
+
+			// eslint-disable-next-line sitekit/acronym-case
+			registry.dispatch( STORE_NAME ).receiveGetProperties( properties, { accountID } );
+			registry.dispatch( STORE_NAME ).receiveGetProfiles( profiles, { accountID, propertyID } );
+
+			registry.dispatch( STORE_NAME ).receiveGetSettings( { accountID } );
```

Comment:
We need to select this account instead of loading it as part of the module settings:
```suggestion
			registry.dispatch( STORE_NAME ).selectAccount( { accountID } );
```

Reply:
@eugene-manuilov If i make that change, then the UI will be different from what we have in Figma + there's a console error.
![image](https://user-images.githubusercontent.com/31818/119645240-e08c5b80-be2e-11eb-9a19-fc8c3ba39e2b.png)

Reply:
@asvinb see an example here: https://github.com/google/site-kit-wp/pull/3421/files#diff-f9c7297e7f2970b10e8c61bab07940db10fd1b64f72c07b1b690e91d3adbc1f6R34-R54

Reply:
@eugene-manuilov Story updated as per the designs in Figma. Actually I didn't want it to match a property so that we could test when the notice would show, i.e after selecting a property. :-)